rtrlib/rtr/packets.c

```diff
@@ -168,6 +191,37 @@ static inline enum pdu_type rtr_get_pdu_type(const void *pdu)
     return *((char *) pdu + 1);
 }
 
+static int rtr_check_interval_range(uint32_t interval, const int interval_type)
```

Comment:
move this function to `rtrlib/rtr.c` to use it in other code parts as well, see below.